--- origsrc/adns-python-1.2.1/adnsmodule.c	2007-01-27 13:09:26.000000000 -0600
+++ src/adns-python-1.2.1/adnsmodule.c	2008-06-24 19:43:37.343750000 -0500
@@ -727,7 +727,7 @@
 ;
 
 static PyTypeObject ADNS_Statetype = {
-	PyObject_HEAD_INIT(&PyType_Type)
+	PyObject_HEAD_INIT(NULL)
 	0,				/*ob_size*/
 	"ADNS_State",			/*tp_name*/
 	sizeof(ADNS_Stateobject),		/*tp_basicsize*/
@@ -938,7 +938,7 @@
 ;
 
 static PyTypeObject ADNS_Querytype = {
-	PyObject_HEAD_INIT(&PyType_Type)
+	PyObject_HEAD_INIT(NULL)
 	0,				/*ob_size*/
 	"ADNS_Query",			/*tp_name*/
 	sizeof(ADNS_Queryobject),		/*tp_basicsize*/
